src/main/java/nextstep/subway/line/domain/Fee.java

```diff
@@ -0,0 +1,74 @@
+package nextstep.subway.line.domain;
+
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Fee {
+    private static final int DEFAULT_FEE = 1250;
+
+    private int fee;
+
+    protected Fee() {}
+
+    private Fee(int fee) {
+        validate(fee);
+        this.fee = fee;
+    }
+
+    private Fee(int defaultFee, int overFare) {
+        validate(overFare);
+        this.fee = defaultFee + overFare;
+    }
+
+    public static Fee of(int fee) {
+        return new Fee(fee);
+    }
+
+    public static Fee ofWithOverFare(int overFare) {
+        return new Fee(DEFAULT_FEE, overFare);
```

Comment:
```suggestion
        return new Fee(DEFAULT_FEE + overFare);
```
이렇게 쓰면 생성자를 하나 줄일 수 있겠네요

Reply:
맞습니다, 
변명이지만😭 너무 늦은시간에 붙잡고 있어서 생각할 수 있는것들도 놓친게 많이 보여서 아쉽네요 :(

피드백 감사합니다 현구님🙏🙏